src/lib/annotations/drawing/DrawingThread.js

```diff
@@ -96,55 +112,65 @@ class DrawingThread extends AnnotationThread {
     //--------------------------------------------------------------------------
 
     /**
-     * Set the drawing styles
+     * Set the drawing styles for a provided context. Sets the context of the memory context if
+     * no other context is provided.
      *
      * @protected
      * @param {Object} config - The configuration Object
      * @param {number} config.scale - The document scale
      * @param {string} config.color - The brush color
+     * @param {CanvasContext} [otherContext] - Optional context to be set
```

Comment:
The naming of otherContext is a bit confusing. Is it a new canvas context from the new drawing or the memory context? I'm assuming this.drawingContext is the memory context so might make sense to rename otherContext to something a little descriptive